test/e2e/ps_test.go

```diff
@@ -185,15 +184,17 @@ var _ = Describe("Podman ps", func() {
 
 		result := podmanTest.Podman([]string{"ps", "-a", "--format", "table {{.Names}}", "--filter", "network=container:alpha"})
 		result.WaitWithDefaultTimeout()
-		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
-		Expect(result.OutputToString()).To(ContainSubstring("bravo"))
+
+		actual := result.OutputToString()
+		Expect(actual).To(ContainSubstring("bravo"))
+		Expect(actual).To(ContainSubstring("NETWORKS"))
 	})
 
 	It("podman ps --filter network=container:<id>", func() {
 		ctrAlpha := "first"
 		container := podmanTest.Podman([]string{"run", "-dt", "--name", ctrAlpha, ALPINE, "top"})
-		container.WaitWithDefaultTimeout()
+		container.WaitWithTimeout(DefaultWaitTimeout * 2)
```

Comment:
You want this test to wait 180 seconds? Why

Reply:
yeah, this might make the CI very unhappy....

Reply:
@rhatdan Several of the tests had .WaitWithDefaultTimeout() listed twice, I added .WaitWithTimeout() so it was clearer what the intention of the tests.